docs/queries/filters.md

```diff
@@ -173,7 +173,7 @@ As well as the date-related searches above, these filters search other propertie
 
 - `tags (include|do not include) <tag>` (Alternative grammar `tag (includes|does not include) <tag>` matching description syntax)
   - Matches case-insensitive (disregards capitalization).
-  - Disregards the global filter when matching.
+  - Disregards the global filter when matching _unless the tasks's global filter includes subtags_.
```

Comment:
Sorry, I don't understand this either. Please could it give a short example? Or be more specific about the behaviour it is describing?

Reply:
Maybe there is a clearer way to say this 😅 here is what I mean...

If I have a set of tasks and a query as shown:
<img width="450" alt="image" src="https://user-images.githubusercontent.com/8586879/180624603-b60cafcc-ff5a-4694-ae37-2c0f204b5677.png">

The results of that query look like this:
<img width="450" alt="image" src="https://user-images.githubusercontent.com/8586879/180624614-ef35eadd-c2cb-4320-9a41-99b4440a0843.png">

It _includes_ any tasks where `sub` is in the global filter. It has always behaved that way (below is a screenshot of the same tasks and query on tasks v1.10.0.
<img width="450" alt="image" src="https://user-images.githubusercontent.com/8586879/180624795-87c4fb52-93be-4ec8-bb8d-f21137804679.png">

Reply:
In exploring this further, I did discover one "breaking" behavior! Specifically, if you query for the text that is your global filter you get:

**On V 1.10.0:**
<img width="450" alt="image" src="https://user-images.githubusercontent.com/8586879/180624898-3fdc71c6-8aef-4792-8efe-ac8fdb392eb3.png">

**This PR:**
<img width="450" alt="image" src="https://user-images.githubusercontent.com/8586879/180624901-e0332ac6-d792-46e9-bff3-3b83a8d8a8e5.png">

Probably something I should address.

Reply:
Thanks very much.

1. I think it would be better to not change the search behaviour in this PR, and make it solely about improvement the dialog...

2. But in the screenshots above, it would be helpful to know what settings you had for these two options:

![image](https://user-images.githubusercontent.com/4840096/180625044-6010a9a0-3f6d-4b74-887b-60e5811fd436.png)